15_Ansible_test/ansible_roles_test/roles/ntp_install/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+        - name: Update all software
+          yum:
+              name: '*'
+              state: latest
+        - name: Install the nginx, mysql
+          yum:
+              name: 
+               - epel-release
+               - nginx
+               - mysql
+               - python3
+              state: latest  
+        - name: Install nodejs
+          yum:
+              name: nodejs
+              state: latest
+        - name: Replace ntp default config
+          template:
+              src: files/nginx.conf
+              dest: /etc/nginx/nginx.conf
```

Comment:
it's good practice to name jinja2 templates with the `.j2.conf`

Reply:
By the way, why you are using the template module for static files?

Reply:
Thanks, I thought It was a good idea to use it for delivery my configure file to the server destination.